src/Compilers/Core/Portable/WellKnownMember.cs

```diff
@@ -431,6 +431,11 @@ internal enum WellKnownMember
         System_Runtime_CompilerServices_TupleElementNamesAttribute__ctorTransformNames,
 
         System_String__Format_IFormatProvider,
+        System_String__op_Implicit_ToReadOnlySpanOfChar,
+
+        System_String__Concat_ReadOnlySpanReadOnlySpan,
```

Comment:
>System_String__Concat_ReadOnlySpanReadOnlySpan [](http://example.com/codeflow?start=8&length=46)

These are lengthy and it takes some effort to figure out how many parameter are there. Consider using a number of parameters in the name. Something like: "System_String__Concat_Of_2_ReadOnlySpans", "System_String__Concat_Of_3_ReadOnlySpans" and "System_String__Concat_Of_4_ReadOnlySpans"